src/test/java/com/google/devtools/build/lib/analysis/ConfigurableAttributesTest.java

```diff
@@ -1279,4 +1279,71 @@ public void multipleMatchErrorWhenAliasResolvesToSameSetting() throws Exception
             + " //a:foo\n"
             + " //a:alias_to_foo");
   }
+
+  @Test
+  public void defaultVisibilityConfigSetting() throws Exception {
+    // Production builds default to private visibility, but BuildViewTestCase defaults to public.
+    setPackageOptions("--default_visibility=private");
+    scratch.file(
+        "c/BUILD", "config_setting(", "    name = 'foo',", "    define_values = { 'foo': '1' })");
```

Comment:
Since the formatter removed your newlines, make this one string.

Reply:
Done.